messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_9_0.xsd

```diff
@@ -0,0 +1,996 @@
+
+            <xs:element name="connector" minOccurs="0" maxOccurs="unbounded" type="connectorType" />
+
+            <xs:element name="http-acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="upgrade-legacy" type="xs:boolean" use="optional" />
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="http-listener" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="remote-acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="socket-binding" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="in-vm-acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="server-id" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="factory-class" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+
+            <xs:element name="jgroup-broadcast-group" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="jgroups-stack" type="xs:string" use="optional">
+                        <xs:annotation>
+                            <xs:documentation>Deprecated. Use jgroups-channel instead.</xs:documentation>
+                        </xs:annotation>
+                    </xs:attribute>
+                    <xs:attribute name="jgroups-channel" type="xs:string" use="optional" />
```

Comment:
as `jgroups-broadcast-group` is a new resource, we may simplify the name of the attributes and get rid of the `jgroups-` prefix for them (that applies for the management attributes and the XSD attributes)

Reply:
Well I wanted to avoid attribute translation. The old methods are still working so they would use the old attribute names. But of course we could have such a Map